### ISSUE TITLE
mkl build error

### ISSUE THREAD

**Issue by armando-fandango** [NervanaSystems/neon#430]
mkl root: /mnt/disk1tb/projects/github/neon/./build/
make[1]: Entering directory '/mnt/disk1tb/projects/github/neon/neon/backends/mklEngine'
make[1]: Leaving directory '/mnt/disk1tb/projects/github/neon/neon/backends/mklEngine'
make[1]: Entering directory '/mnt/disk1tb/projects/github/neon/neon/backends/mklEngine'
Building mklEngine.so...
gcc src/conv.c src/pooling.c src/relu.c src/batchNorm.c src/concat.c src/softmax.c src/MKLDNN.h -shared -o mklEngine.so -std=c99 -O3 -I/mnt/disk1tb/projects/github/neon/./build//include -L/mnt/disk1tb/projects/github/neon/./build//lib -Wl,-rpath=/mnt/disk1tb/projects/github/neon/./build//lib -fopenmp -lmklml_gnu -fPIC -march=native -g -liomp5
In file included from src/conv.c:15:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
In file included from src/pooling.c:15:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
In file included from src/relu.c:15:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
In file included from src/batchNorm.c:15:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
In file included from src/concat.c:15:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
In file included from src/softmax.c:16:0:
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
src/MKLDNN.h:23:21: fatal error: mkl_dnn.h: No such file or directory
compilation terminated.
Makefile:40: recipe for target 'mklEngine.so' failed
make[1]: *** [mklEngine.so] Error 1
make[1]: Leaving directory '/mnt/disk1tb/projects/github/neon/neon/backends/mklEngine'
Installing neon dependencies system wide...

**Comment by baojun-nervana** [comment]
@armando-fandango Would you reference to https://github.com/NervanaSystems/neon/issues/428?

You may try

make clean
rm -rf build mklml_lnx_*
make
